### PR TITLE
use exchange rates in merchant payments to display appropriate amount of SPEND + validate min amount

### PR DESCRIPTION
**Pull request by Aierie**
- [x] Add caching?
- [x] Try out with the merchant payment request page
- [x] Documentation for new env vars
- [x] Get API key, update fetching code to accept base=USD
- [ ] add secret, communicate to others for dev purposes

### CODE REVIEW

File: packages/web-client/tests/acceptance/pay-test.ts [hunk 1]
```diff
@@ -287,6 +297,59 @@ module('Acceptance | pay', function (hooks) {
...
+    let expectedUrl = generateMerchantPaymentUrl({
+      domain: universalLinkDomain,
+      network,
+      merchantSafeID: merchantSafe.address,
+      currency: jpySymbol,
+      amount: jpyAmount,
+    });
+    assert.dom(QR_CODE).hasAttribute('data-test-styled-qr-code', expectedUrl);
+    assert.dom(PAYMENT_URL).containsText(expectedUrl);
+  });
```
Comment (backspace) [inline packages/web-client/tests/acceptance/pay-test.ts:351]: I’m glad to see this explicit handling of a failure fetching exchange rates, should there be some assertion that indicates how the UI differs in this case though? Like at least `assert.dom(SECONDARY_AMOUNT).doesNotExist()`

**Review by backspace — APPROVED** [review]
If we end up needing to proxy more APIs that don’t provide origin-filtering we might find ways to generalise this, but this looks good for exchange rates/Fixer at least 🎉

File: packages/hub/services/exchange-rates.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+   *     KRW: 1370.453432,
+   *     RUB: 84.171385,
+   *     INR: 85.915208,
+   *     JPY: 129.027569,
+   *     TRY: 10.291685,
+   *     CAD: 1.468866,
+   *     NZD: 1.679484,
+   *     ZAR: 17.450637
+   *   }
+   * }
```
[the remark this answers is not in the document]
Comment (Aierie) [reply packages/hub/services/exchange-rates.ts:59]: Yeah... this was an EUR response. I guess the numbers will appear off, I can replace this with a USD response.

File: packages/hub/services/exchange-rates.ts [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        if (result?.success) {
+          this.cachedValue = result;
+          this.lastFetched = result.timestamp * 1000;
+        } else {
+          Sentry.captureException(result ?? 'Fetching exchange rates returned a falsey value');
+          this.invalidateCache();
+        }
+        return result;
+      } catch (e) {
+        Sentry.captureException(e ?? 'Fetching exchange rates returned a falsey value');
```
[the remark this answers is not in the document]
Comment (Aierie) [reply packages/hub/services/exchange-rates.ts:90]: You're right, will correct this.

File: packages/web-client/tests/acceptance/pay-test.ts (hunk 1 above)
Comment (Aierie) [reply packages/web-client/tests/acceptance/pay-test.ts:352]: I think I accidentally removed that. Thanks!

**Review by lukemelia — APPROVED** [review]

**Pull request merged**